### ISSUE TITLE
Upgrade scalaparse dependency.  java.lang.ClassNotFoundException: fastparse.Utils$FuncName

### ISSUE THREAD

**Issue by olafurpg** [lihaoyi/Scalatex#25]
I have a scalatex 0.3.5 `readme` project that transitively depends on scalaparse 0.3.7 via a `core` project. This means the scalaparse dependency in `readme` is overriden to 0.3.7 (instead of 0.3.1, which scalatex requires). This causes the following error during `readme/run`:

``` scala
[error] /Users/ollie/dev/scala/scalafmt/readme/target/scala-2.11/src_managed/main/scalatex/Readme.scala:7: exception during macro expansion:
[error] java.lang.ClassNotFoundException: fastparse.Utils$FuncName
[error]     at java.net.URLClassLoader$1.run(URLClassLoader.java:372)
[error]     at java.net.URLClassLoader$1.run(URLClassLoader.java:361)
[error]     at java.security.AccessController.doPrivileged(Native Method)
[error]     at java.net.URLClassLoader.findClass(URLClassLoader.java:360)
[error]     at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
[error]     at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
[error]     at scalatex.stages.Parser.TextNot(Parser.scala:44)
[error]     at scalatex.stages.Parser.<init>(Parser.scala:50)
[error]     at scalatex.stages.Parser$.apply(Parser.scala:17)
[error]     at scalatex.stages.Parser$.apply(Parser.scala:15)
[error]     at scala.Function2$$anonfun$tupled$1.apply(Function2.scala:54)
[error]     at scala.Function2$$anonfun$tupled$1.apply(Function2.scala:53)
[error]     at scalatex.package$Internals$.compile$1(package.scala:84)
[error]     at scalatex.package$Internals$.compileThing(package.scala:101)
[error]     at scalatex.package$Internals$.applyMacroFileBase(package.scala:43)
[error]     at scalatex.package$Internals$.applyMacroFile(package.scala:27)
[error]   def apply(): Frag = _root_.scalatex.twf("/Users/ollie/dev/scala/scalafmt/readme/Readme.scalatex")
```

`FuncName` was removed in [this commit](https://github.com/lihaoyi/fastparse/commit/98eced77bbf366423c02c27aa20cc75916617262#diff-57ac1264e9f4624aa480030d9f4fd0de).

FWIW, adding 

``` scala
dependencyOverrides += "com.lihaoyi" %% "scalaparse" % "0.3.1"
```

to the `readme` settings solves the problem in my case, at least for now.

**Comment by xeno-by** [comment]
This is now fixed in 0.3.7.

**Comment by olafurpg** [comment]
Scalatex is now using the latest fastparse version.
